src/intents.cs

```diff
@@ -2357,6 +2340,25 @@ interface INRestaurantReservationUserBooking : NSCopying {
 
 	[Introduced (PlatformName.iOS, 10, 0)]
 	[Unavailable (PlatformName.MacOSX)]
+	[DisableDefaultCtor]
+	[BaseType (typeof (INIntentResolutionResult))]
+	interface INRestaurantResolutionResult {
+
+		[Static]
+		[Export ("successWithResolvedRestaurant:")]
+		INRestaurantResolutionResult GetSuccess (INRestaurant resolvedRestaurant);
```

Comment:
Same as before: "If https://github.com/xamarin/xamarin-macios/pull/555/files#r73679252 turns out to be the way to go, we should apply the same thing for the 3 methods here."